translator/translate/otel/pipeline/applicationsignals/translator_test.go

```diff
@@ -132,6 +132,28 @@ func TestTranslatorMetricsForKubernetes(t *testing.T) {
 			detector:   eksdetector.TestEKSDetector,
 			isEKSCache: eksdetector.TestIsEKSCacheEKS,
 		},
+		"WithAppSignalsAndLoggingEnabled": {
+			input: map[string]interface{}{
+				"agent": map[string]interface{}{
+					"debug": true,
+				},
+				"logs": map[string]interface{}{
+					"metrics_collected": map[string]interface{}{
+						"app_signals": map[string]interface{}{
```

Comment:
we should use `application_signals` as the new official config

Reply:
Not sure how this passed unit testing without that change.

Reply:
we allow for both app_signals and application_signals, but after speaking with the pulse team they are okay with deprecating app_signals but we haven't made that change